README.adoc

```diff
@@ -141,28 +140,28 @@ include::finish/src/main/java/io/openliberty/guides/system/SystemHealth.java[tag
 
 The `org.eclipse.microprofile` package contains the required MicroProfile Health Check classes.
 
-The `@Health` annotation indicates that this class is read to report the state of
-a service to the `http://localhost:9080/health` endpoint. The `@ApplicationScoped` annotation is
+The `@Health` annotation indicates that this class will be used to report the state of
```

Comment:
I think ID does not like future passive voice. "will be used to report" => "reports"

Reply:
Fixed